liquidhaskell-boot/src/Language/Haskell/Liquid/Transforms/CoreToLogic.hs

```diff
@@ -485,15 +485,25 @@ brels = M.fromList [ (symbol ("==" :: String), Eq)
 
 bops :: M.HashMap Symbol Bop
 bops = M.fromList [ (numSymbol "+", Plus)
+                  , (numIntSymbol "+", Plus)
```

Comment:
It could be a good time to include a comment spelling what you have learned about bops, which motivates listing these operators here.

Reply:
Done in a8a111a42c324387b8cea0c9cd20b2295d0ec83d